utils/keybinding.js

```diff
@@ -37,7 +37,7 @@ exports.hasKeybinding = function (key) {
 
 function keyupHandler (matchChar, callback, evt) {
   // we don't want to fire events when somebody is typing in an input, esc is okay to fire
-  if ((evt.target.tagName !== 'INPUT' && evt.target.tagName !== 'TEXTAREA') || evt.keyCode === KEY_NAMES.esc) {
+  if ((evt.target.tagName !== 'INPUT' && evt.target.tagName !== 'TEXTAREA' && !evt.target.hasAttribute('data-medium-editor-element')) || evt.keyCode === KEY_NAMES.esc) {
```

Comment:
don't reference the data-medium-editor, that could change and not everybody uses that component